pkg/store/schema.go

```diff
@@ -33,14 +35,15 @@ CREATE TABLE IF NOT EXISTS validators (
 	id SERIAL PRIMARY KEY,
 	public_key BYTEA NOT NULL UNIQUE,
 	address BYTEA NOT NULL UNIQUE,
+	name TEXT,
 	memo TEXT
 );
 
 ---
 
 CREATE TABLE IF NOT EXISTS blocks (
 	block_height BIGINT NOT NULL PRIMARY KEY,
-	block_hash BYTEA NOT NULL,
+	block_hash TEXT NOT NULL UNIQUE,
```

Comment:
I had to change these to text. The guy we employed to make front end was having a very hard time understanding what bytes are. I made these changes to comfort him 💃